defi/src/protocols/data3.ts

```diff
@@ -40253,7 +40253,7 @@ const data3: Protocol[] = [
     module: "avalon-finance/index.js",
     twitter: "avalonfinance_",
     github: ["avalonfinancexyz"],
-    audit_links: ["https://github.com/DefiLlama/defillama-server/pull/7838"],
+    audit_links: ["https://drive.google.com/file/d/1NQKM8C9rVHuplDqiDMdwKykLA4cUQbUj"],
```

Comment:
no google drive files

Reply:
We have updated the link to Salus Audit Reports (https://audit.salusec.io/):
- https://cert-api.salusec.io/api/v1/salus/contract/certificate/full/2024/Avalon_Finance-ORACLE_audit_report_2024-04-09.pdf
- https://cert-api.salusec.io/api/v1/salus/contract/certificate/full/2024/Avalon-AVAF_audit_report_2024-06-11.pdf